### DISCUSSION TITLE
Why is 'log' keyword in the return parameter of training_step of LightningModule is not working?

### DISCUSSION THREAD

**Discussion by kochark1** [Lightning-AI/pytorch-lightning#13115] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I am seeing the following message in the Tensorboard:
    **No dashboards are active for the current data set.** 

I use the following code

        def training_step(self, batch, batch_idx):
                
                # computes loss
        
                tensorboard_logs = {'train_loss': loss}
        
                return {"loss": loss, 'log': tensorboard_logs}

I do the same in validation_epoch_end() as well.

Is this functionality of logging through the 'log' keyword not supported anymore? Should we log explicitly using self.log() or something similar?

**Comment by akihironitta** [comment, marked as answer]
@kochark1 No, it isn't supported since 1.0.0 (#3681), and yes, you need to use `self.log(...)`. See details here: https://pytorch-lightning.readthedocs.io/en/stable/extensions/logging.html

**Comment by danielyan86129** [comment]
saved my day...
